installation_and_upgrade/ibex_install_utils/tasks/backup_tasks.py

```diff
@@ -43,35 +43,36 @@ def zip_file(self, filename, zipname):
                     zipf.write(os.path.join(root, file), arcname=os.path.join(root, file).replace(filename, ''))
                     self.update_progress_bar(i, len(all_files))
                     i = i + 1
+                    
+    def move_file(self, src, dst, copy=False, ignore=None):
```

Comment:
I think the name of this function is potentially confusing, it doesn't move a file, it either moves or copies files between the directories passed. Maybe calling it `backup_files` would reflect its true functionality better?

Reply:
Yes it is confusing, it must've developed from originally just moving a file to what it is now.